deepcell_toolbox/utils.py

```diff
@@ -279,6 +279,8 @@ def resize(data, shape, data_format='channels_last'):
             Batch and channel dimensions are handled automatically and preserved.
         data_format (str): determines the order of the channel axis,
             one of 'channels_first' and 'channels_last'.
+        data_type (str): determines whether resizing raw data or labels
+            one of 'X' or 'y'
```

Comment:
Would it be clearer to have the user specify int or float instead of assuming the dtype of X and y? @willgraf

Reply:
That's actually what I had initially!

However, I think if we have raw image data stored as ints, we still want anti-aliasing: we don't care if novel pixel values get introduced, it's better to have smooth inputs for the model. 

At least that was my rationale, and also the reason we can't just rid of the parameter completely and have resize make the determination just based on the dtype.

Reply:
I don't understand why we need a new input for the data type. Can't we tell what the data type is by just using `data.dtype`?

I want to push back on having the string parameter, I don't think it's flexible enough.

Reply:
Okay, let's say we have some IF data. This data is stored as int16. We want to do regular resizing, using interpolation, on this data, even though it's an int, not a float. 

We might separately have labeled data. This will also be stored as int16. We don't want to do the regular resizing with this data, we want to use this alternate functionality that I've added here. 

So just by looking at the dtype, we don't know if it's raw images or labeled data.

Reply:
OK, that makes sense. But I think we can improve the logic of `data_type` parameter. We only have logic that checks if `data_type == 'X'`, so maybe we can have this parameter be a boolean value that both sets the order and turns on/off anti-aliasing.

I see no reason for this to be a string value, especially since if it is anything other than `"X"`, we don't care.